MANIFEST.in

```diff
@@ -9,6 +9,7 @@ recursive-include hnn_core/mod/x86_64 *
 recursive-include hnn_core/mod/x86_64/.libs *
 
 recursive-include hnn_core *.py
+recursive-include hnn_core *.ipynb
```

Comment:
What exactly is `MANIFEST.in` for again? I'm just trying to follow the logic here...

Reply:
whatever files are included in MANIFEST.in will be copied into the anaconda directory when the install is done. I put this in explicitly in case ipynb files were not copied by default.

Reply:
I see, thanks!